aten/src/ATen/native/README.md

```diff
@@ -170,6 +170,10 @@ Note that argument type modifiers such as defaults and optional are not currentl
 
 The declarations also support the following attributes:
 
+**Overloads.** You can register multiple functions with the same name and different
```

Comment:
it renders weirdly (there's a preview button on the top right) as it's in the middle of the list of "attributes"

Reply:
this is an improvement, but it still leaves some unanswered questions.  For example: do I need to provide an overload if my function has the same name?  Is there a standard naming rule for the overload?

Reply:
we don't have a fixed naming scheme defined yet, apart from "try to find a good, semantically meaningful name". Is your other question whether overload names need to be present and unique if the same function name appears multiple times? The text says "you can register multiple functions [...] *if* you give them unique overload names". So the answer would be yes. Should I change that to "only if"?

Reply:
I think I'd be looking for more guidance.  It's not clear to be what I _should_ do, even if it says what is possible to do.

Reply:
I added some more text. Is this better?

Reply:
@dzhulgakov not sure what you mean with the rendering issue. Just checked and at least now (after making the text longer), I don't find what you mean. Is it fixed or still there?

Reply:
It's not rendering per-se, just weird that "The declarations also support the following attributes:" starts a list of attributes (with ### header), but the first thing is not an item in the list, but just a bold text. Maybe just reorder this paragraph above the previous one?